### PR TITLE
MWPW-120782 - Fix Chart Color Order

### PR DESCRIPTION
**Pull request by meganthecoder**
* Fix the Chart block color palette to match Spectrum's order

Resolves: [MWPW-120782](https://jira.corp.adobe.com/browse/MWPW-120782)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/methomas/chart-spectrum-colors?martech=off
- After: https://methomas-chart-spectrum-colors--milo--adobecom.hlx.page/drafts/methomas/chart-spectrum-colors?martech=off
- After: https://main--bacom--adobecom.hlx.page/resources/holiday-shopping-report?milolibs=methomas-chart-spectrum-colors

### CODE REVIEW

File: test/blocks/chart/chart.test.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ describe('chart', () => {
 
   it('getColors returns rotated color list if color provided ', () => {
     const authoredColor = 'indigo';
-    const colors = ['#4046CA', '#7326D3', '#147AF3', '#72E06A', '#7E84FA', '#DE3D82', '#008F5D', '#CB5D00', '#E8C600', '#BCE931', '#0FB5AE', '#F68511'];
+    const colors = ['#4046CA', '#F68511', '#DE3D82', '#7E84FA', '#72E06A', '#147AF3', '#7326D3', '#E8C600', '#CB5D00', '#008F5D', '#BCE931', '#0FB5AE'];
```
Comment (JasonHowellSlavin) [inline test/blocks/chart/chart.test.js:109]: nit. What about:

```
import { colorPalette } from 'chart.js';

const colors = Object.values(colorPalette);
```

That way you could only have them in one place?

**Review by JasonHowellSlavin — APPROVED** [review]

**Review by chrischrischris — APPROVED** [review]

File: test/blocks/chart/chart.test.js (hunk 1 above)
Comment (meganthecoder) [reply test/blocks/chart/chart.test.js:109]: `colors` is not the same as `colorPalette`. It's a rotated color palette starting with indigo (which isn't the first color).

Comment (JasonHowellSlavin) [reply test/blocks/chart/chart.test.js:109]: I see. I guess the core data is the same though, right, and nearly in the correct order? 

They may never ask to change them again, but you could do something like. 
```
const colors = Object.values(colorPalette);
const firstColor = colors.shift();
colors.push(firstColor);
```
🤷  up to you. 

Not a blocker, just a thought.

**Review by JasonHowellSlavin — APPROVED** [review]

File: test/blocks/chart/chart.test.js (hunk 1 above)
Comment (meganthecoder) [reply test/blocks/chart/chart.test.js:109]: Thanks for the suggestion but I prefer it hardcoded. That way I know that the expected value (colors) is exactly what I want. If it's generated by code, there's a small chance the code in the test could be broken/wrong and we get a false positive.

**Pull request merged**